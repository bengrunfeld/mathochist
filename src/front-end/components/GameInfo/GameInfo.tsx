import React from 'react'

import PlayerName from '../PlayerName'
import GameStats from '../GameStats'

const GameInfo = () => (
  <div className='row'>
    <PlayerName />
    <GameStats />
  </div>
)

export default GameInfo
