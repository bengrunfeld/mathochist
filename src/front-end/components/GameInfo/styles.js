import styled from 'styled-components'

const GameInfoContainer = styled.div`
  margin-top: 10px;
`

export { GameInfoContainer }
