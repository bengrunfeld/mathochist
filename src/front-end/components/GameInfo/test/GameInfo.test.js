import React from 'react'
import Enzyme, { shallow, mount, render } from 'enzyme'
import { MemoryRouter } from 'react-router-dom'

import GameInfo from '../GameInfo'

describe('<GameInfo />', () => {
  test('matches the snapshot', () => {
    const tree = mount(<GameInfo />)
    expect(tree).toMatchSnapshot()
  })
})
