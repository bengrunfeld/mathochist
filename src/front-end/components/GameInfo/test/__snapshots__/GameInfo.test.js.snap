// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GameInfo /> matches the snapshot 1`] = `
<GameInfo>
  <div>
    <h1>
      GameInfo
    </h1>
  </div>
</GameInfo>
`;
